tests/templates/kuttl/cluster-operation/30-assert.yaml.j2

```diff
@@ -0,0 +1,43 @@
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+timeout: 600
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: hdfs-namenode-default
+status:
+  availableReplicas: 0
+  replicas: 0
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: hdfs-journalnode-default
+status:
+  availableReplicas: 0
+  replicas: 0
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: hdfs-datanode-default
+status:
+  availableReplicas: 0
+  replicas: 0
+---
+apiVersion: v1
+kind: ConfigMap
```

Comment:
I would remove the ConfigMap asserts

Reply:
https://github.com/stackabletech/hdfs-operator/pull/337/commits/79c69a7b83bf498846d7b50de231e88ef4196091